weed/stats/metrics_names.go

```diff
@@ -4,6 +4,8 @@ package stats
 // The naming convention is ErrorSomeThing = "error.some.thing"
 const (
 	// volume server
+	LocalWrite                  = "localWrite"
```

Comment:
use "writeToLocalDisk" to align with "errorWriteToLocalDisk".
use "writeToReplicas" to align with "errorWriteToReplicas".

Reply:
> use "writeToLocalDisk" to align with "errorWriteToLocalDisk". use "writeToReplicas" to align with "errorWriteToReplicas".

What does align mean?

Reply:
these are constant values in line 14-17 of this file.